src/main/resources/application.yaml

```diff
@@ -23,8 +23,14 @@ spring:
     allow-bean-definition-overriding: true
   servlet:
     multipart:
-      max-file-size: 100MB
-      max-request-size: 100MB
+      max-file-size: 500MB
+      max-request-size: 500MB
+
+logging:
+  level:
+    root: info
```

Comment:
for the purpose of the debugging doc issue shall we set this to debug for now? and the one below?